lib/qsim_server.rb

```diff
@@ -3,7 +3,12 @@
 Mumukit.runner_name = 'qsim'
 Mumukit.configure do |config|
   config.docker_image = 'mumuki/mumuki-qsim-worker'
+  config.content_type = 'html'
 end
 
+require 'active_support/all'
```

Comment:
it should not be necessary to do this require, since mumukit already requires it